src/importnode.h

```diff
@@ -27,7 +27,7 @@ class ImportNode : public LeafNode
 	std::string layername;
 	int convexity;
 	double fn, fs, fa;
-	double origin_x, origin_y, scale;
+	double origin_x, origin_y, center, scale;
```

Comment:
why is center a double and not a bool? 

Cross checking with other parts of openscad
https://github.com/openscad/openscad/blob/707043efe658e9bf41ea102fafad1c093c7fdbfd/src/linearextrudenode.h#L23
https://github.com/openscad/openscad/blob/707043efe658e9bf41ea102fafad1c093c7fdbfd/src/primitives.cc#L102
bool seams tobe what we usually use.

Reply:
see other comment :).